### PR TITLE
PPM-383: remove unsupported 60ghz option from the core

### PR DESCRIPTION
**Pull request by DanielEdriPurpl**
Remove from the persistent DB all references to the unsupported 60GHz band.
This includes the BML CLI, the DB, and anything in between.

### CODE REVIEW

File: framework/tlvf/AutoGenerated/include/tlvf/WSC/eWscRfBands.h [hunk 1]
```diff
@@ -19,13 +19,8 @@
...
-    WSC_RF_BAND_5GHZ = 0x2,
-    WSC_RF_BAND_60GHZ = 0x4,
-};
+enum eWscRfBands : uint8_t { WSC_RF_BAND_2GHZ = 0x1, WSC_RF_BAND_5GHZ = 0x2 };
 
-
-}; // close namespace: WSC
+}; // namespace WSC
 
 #endif //_TLVF/WSC_EWSCRFBANDS_H_
```
Comment (itayx) [inline framework/tlvf/AutoGenerated/include/tlvf/WSC/eWscRfBands.h]: The removal need to be only in the persistent DB

File: framework/tlvf/yaml/tlvf/WSC/eWscRfBands.yaml [hunk 2]
```diff
@@ -7,5 +7,3 @@ eWscRfBands:
   _enum_storage: uint8_t
   WSC_RF_BAND_2GHZ: 0x01
   WSC_RF_BAND_5GHZ: 0x02
-  WSC_RF_BAND_60GHZ: 0x04
-
```
Comment (itayx) [inline framework/tlvf/yaml/tlvf/WSC/eWscRfBands.yaml]: The removal need to be only in the persistent DB

File: framework/tlvf/yaml/tlvf/ieee_1905_1/tlvSupportedFreqBand.yaml [hunk 3]
```diff
@@ -16,4 +16,3 @@ eValue:
   _enum_storage: uint8_t
   BAND_2_4G: 0x00
   BAND_5G: 0x01
-  BAND_60G: 0x02
```
Comment (itayx) [inline framework/tlvf/yaml/tlvf/ieee_1905_1/tlvSupportedFreqBand.yaml]: The removal need to be only in the persistent DB

File: controller/src/beerocks/master/son_master_thread.cpp [hunk 4]
```diff
@@ -430,10 +430,6 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_search(const std::string
         tlvSupportedFreqBand->value() = ieee1905_1::tlvSupportedFreqBand::BAND_5G;
         break;
     }
-    case ieee1905_1::tlvAutoconfigFreqBand::IEEE_802_11_60_GHZ: {
-        tlvSupportedFreqBand->value() = ieee1905_1::tlvSupportedFreqBand::BAND_60G;
-        break;
-    }
```
Comment (itayx) [inline controller/src/beerocks/master/son_master_thread.cpp]: The removal need to be only in the persistent DB

File: agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp [hunk 5]
```diff
@@ -2874,9 +2874,6 @@ bool backhaul_manager::handle_1905_autoconfiguration_response(ieee1905_1::CmduMe
             band_name = "5GHz";
             freq_type = beerocks::eFreqType::FREQ_5G;
             break;
-        case ieee1905_1::tlvSupportedFreqBand::BAND_60G:
-            LOG(DEBUG) << "received autoconfiguration response for 60GHz band, unsupported";
-            return true;
```
Comment (itayx) [inline agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp]: The removal need to be only in the persistent DB

**Review by itayx — CHANGES_REQUESTED** [review]
Two things:
1) The removal of the 60GHz band needs to be contained to only the Persistent DB feature.
2) Please follow the guidelines of how commit messages are supposed to be structured as described [here](https://github.com/prplfoundation/prplMesh/blob/master/CONTRIBUTING.md#commits)

**Review by itayx — APPROVED** [review]
Looks good, please add PR description

**Review by mariomaz — APPROVED** [review]
LGTM

File: common/beerocks/tlvf/AutoGenerated/include/beerocks/tlvf/beerocks_message_common.h [hunk 6]
```diff
@@ -1342,23 +1342,21 @@ typedef struct sBssidInfo {
     uint32_t downlink_estimated_mac_data_rate_mbps;
     uint32_t uplink_estimated_mac_data_rate_mbps;
     uint8_t sta_measured_uplink_rssi_dbm_enc;
```
Comment (adam1985d) [inline common/beerocks/tlvf/AutoGenerated/include/beerocks/tlvf/beerocks_message_common.h:1344]: suggested commit message:
```
common: tlvf: remove the unsupported option of 60ghz

The 60Ghz band is not supported and should not be part of the possible
configuration of the selected bands.

Remove the 60Ghz from the eSelectedBands enum which is used by the
persistent DB to store the selected bands.

https://jira.prplfoundation.org/browse/PPM-383

Signed-off-by: DanielEdriPurpl <daniel.edri@devalore.com>
```

File: controller/src/beerocks/bml/bml_defs.h [hunk 7]
```diff
@@ -126,7 +126,6 @@ extern "C" {
 #define BML_CLIENT_SELECTED_BANDS_24G 1
 #define BML_CLIENT_SELECTED_BANDS_5G 2
 #define BML_CLIENT_SELECTED_BANDS_6G 4
```
Comment (adam1985d) [inline controller/src/beerocks/bml/bml_defs.h:128]: suggested commit message:
```
bml: remove the unsupported 60ghz option from selected bands

Remove the 60ghz option from the BML_CLIENT_SELECTED_BANDS in the
bml_defs and bml_cli.

https://jira.prplfoundation.org/browse/PPM-383

Signed-off-by: DanielEdriPurpl <daniel.edri@devalore.com>
```

**Review by adam1985d — CHANGES_REQUESTED** [review]
please fix commit messages and will re-review

**Pull request closed without merging**